doc/man3/BN_add.pod

```diff
@@ -92,7 +92,9 @@ BN_exp() raises I<a> to the I<p>-th power and places the result in I<r>
 BN_mul().
 
 BN_mod_exp() computes I<a> to the I<p>-th power modulo I<m> (C<r=a^p %
-m>). This function uses less time and space than BN_exp().
+m>). This function uses less time and space than BN_exp(). Note that calling
```

Comment:
"not supported" doesn't mean much.  Does it return an error?  Perhaps this:
```
Do not call this function when B<m> is even and any of the parameters have the B<BN_FLG_CONSTTIME> flag set.
```

Reply:
Done